### PR TITLE
Rdcc 3612

### PR DESCRIPTION
**Pull request by sahitya-desireddy**
**Before creating a pull request make sure that:**

Added Swagger Configuration

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-3612

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/cdapi/config/SwaggerConfiguration.java [hunk 1]
```diff
@@ -2,25 +2,45 @@
...
+            .apis(RequestHandlerSelectors.withClassAnnotation(RestController.class))
             .paths(PathSelectors.any())
-            .build();
+            .build()
+            .securitySchemes(apiKeyList());
+    }
+
+    private  List<SecurityScheme> apiKeyList() {
+        return
+            newArrayList(
```
Comment (SujeethKumarAri) [inline src/main/java/uk/gov/hmcts/reform/cdapi/config/SwaggerConfiguration.java:39]: You can replace this with List.of()

File: src/main/java/uk/gov/hmcts/reform/cdapi/config/SwaggerConfiguration.java [hunk 2]
```diff
@@ -2,25 +2,45 @@
...
+                new ApiKey("Authorization", "Authorization","header"),
+                new ApiKey("ServiceAuthorization", "ServiceAuthorization", "header"),
+                new ApiKey("UserEmail", "UserEmail", "header")
```
Comment (SujeethKumarAri) [inline src/main/java/uk/gov/hmcts/reform/cdapi/config/SwaggerConfiguration.java:42]: Any reason for adding this header?

**Review by SujeethKumarAri — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/cdapi/config/SwaggerConfiguration.java (hunk 2 above)
Comment (ManishParyani) [reply src/main/java/uk/gov/hmcts/reform/cdapi/config/SwaggerConfiguration.java:42]: Same thing i was about to ask :)

Comment (sahitya-desireddy) [reply src/main/java/uk/gov/hmcts/reform/cdapi/config/SwaggerConfiguration.java:42]: I was not aware of it as i copied it from some other project , if not required will remove it now ...Thank you Guys ...

**Review by SujeethKumarAri — APPROVED** [review]

**Review by ManishParyani — APPROVED** [review]

**Pull request merged**